README.md

```diff
@@ -23,7 +23,7 @@ module.exports = {
   'loaders': [{
     'loader': 'babel-loader',
     'test': /\.js$/,
-    'exclude': /node_modules/,
+    'exclude': '/node_modules/',
```

Comment:
Doh. Sorry about that. I was thinking that it was just an argument that wasn't commented properly. My bad.